simpleLooping/normalizeString.py

```diff
@@ -18,3 +18,26 @@
 #   normalizeString("")                  # return ""
 #   normalizeString("Hello  World!")     # return "Hello World!"
 #   normalizeString("  Hello  World!  ") # return "Hello World!"
+from typing import List
+
+def normalizeString(string: str) -> str:
+    lenLst = len(string)
+
+    binary = 0
```

Comment:
Why use "binary" here? we don't even have any "binary" here at all ... Did  you copy paste? And again, if we sue it as iterable or index then use 'i' as name please.